### DISCUSSION TITLE
Android app sometime suggests app wants to be in Solana Testnet mode

### DISCUSSION THREAD

**Discussion by eoin-betdex** [phantom/docs#197] [Help & Support] [answered]
### Summary

Sometimes when users attempt to connect their wallet on our site in Chrome for Android when the Phantom app launches they get an error from Phantom suggesting the app wants to use Solana Testnet. I don't understand how this occurs. Is there anywhere I can read about when/why this message might appear within Phantom? 

The app is configured to connect to a mainnet rpc endpoint so this is confusing. 

Thanks! Screenshot below of the error within the Phantom Android App. 

![Screenshot_20240312_001619_Phantom](https://github.com/phantom/docs/assets/95632764/1735428c-f3a1-4ba0-ab98-ed27c6fa1e52)

### Example

https://app.betdex.com

### Steps to Reproduce

Load the site. 
Sign up/Sign in.
Press Connect wallet button. 
Phantom launches. 
Error is present. 

### Phantom Version

24.3.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @eoin-betdex,

This means our provider is detecting the testnet is configured as a network option somewhere. I would recommend trying to determine the situation in which users see this error and then investigating the network connection configuration in that state. If you have any code to share, I'm happy to take a look.
